Casks/zoom.rb

```diff
@@ -31,8 +31,17 @@
     # Description: Ensure console variant of postinstall is non-interactive.
     # This is because `open "$APP_PATH"&` is called from the postinstall
     # script of the package and we don't want any user intervention there.
-    if system_command("ps", args: ["x"]).stdout.match?("zoom.us.app/Contents/MacOS/zoom.us")
-      system_command "/usr/bin/pkill", args: ["-f", "#{appdir}/zoom.us.app"], must_succeed: false
+    ohai "The Zoom package postinstall script launches the Zoom app"
+    ohai "Attempting to close zoom.us.app to avoid unwanted user intervention"
+    if system_command("/bin/ps", args: ["x"]).stdout.match?("zoom.us.app/Contents/MacOS/zoom.us")
+      begin
+        retries ||= 3
+        system_command "/usr/bin/pkill", args: ["-f", "#{appdir}/zoom.us.app"]
```

Comment:
As I mentioned in https://github.com/Homebrew/homebrew-cask/issues/107182#issuecomment-860095834, `zoom` uses a package installer that will ignore `appdir` input.

So, this should be fixed to `args: ["-f", "/Applications/zoom.us.app"]` otherwise it will always fail if someone uses sets `--appdir` (either in local command or global environment/homebrew-bundle).

Reply:
Of course, you are correct. As I understand it, Zoom installs to `/Applications` if the user has administrator privileges and to `~/Applications` if not. Since `pkill` without `-x` will, by default, match any substring, it probably makes sense to match only `/zoom.us.app`.
```suggestion
        system_command "/usr/bin/pkill", args: ["-f", "/zoom.us.app"]
```